qcodes/dataset/sqlite_base.py

```diff
@@ -255,8 +256,13 @@ def _convert_numeric(value: bytes) -> Union[float, int, str]:
     # outcome is 'nan'
     if np.isnan(numeric):
         return numeric
+    
+    # If that worked, e.g. did not raise an exception, then we check if the
+    # outcome is 'inf'
+    if np.isinf(numeric):
```

Comment:
What about the case of negative infinity? Shall it also be covered in a similar way? If so, please also add a test for it.